__tests__/routes/applications.test.js

```diff
@@ -181,13 +183,13 @@ describe('Application Router', () => {
       it('returns application(s) for a given user', () => {
         const expected = {
           application_id: 6,
-          created_at: '2022-01-28T23:38:28.256Z',
-          first_name: 'User',
-          last_name: '10',
+          // created_at: '2022-01-28T23:38:28.256Z',
+          // first_name: 'Johnny',
+          // last_name: 'Donuts', //! will not have necessary info if they change their name - test will fail for no reason!
```

Comment:
Why keep these lines of code, 186 to 188?

Reply:
Yes. Please remove these lines.

Reply:
done!